core/assets/bundles/bundle.properties

```diff
@@ -1237,6 +1237,7 @@ rules.buildcostmultiplier = Build Cost Multiplier
 rules.buildspeedmultiplier = Build Speed Multiplier
 rules.deconstructrefundmultiplier = Deconstruct Refund Multiplier
 rules.waitForWaveToEnd = Waves Wait for Enemies
+rules.wavelimit = Wave Limit
```

Comment:
this doesn't make clear that the game ends when the target is hit - maybe call it "Map Ends After Wave"?